recipes/python_abi/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: python_abi
+  version: {{ python_version }}
+
+build:
+  number: 0
+  string: {{ python_abi_tag }}
```

Comment:
The typical drawback applies here: If you'd want to change/refine the dependencies in the future, a new build would have the same filename. You'd have to change the version number to something like `x.y.z.0.custom_number` (w/o relying on repodata patching).

Reply:
Any other suggestions?

Reply:
Doing the typical `python_abi[build=*_abi]` thing, i.e.,
`h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ python_abi_tag }}`
(well `{{ PKG_BUILDNUM }}_{{ python_abi_tag }}` would also work since we don't have build deps).
Sure it's a little less convenient to have the user type `python=3.8.2=*cp38` `python=3.8.2=cp38`, but at least we're more flexible then.

Reply:
Thanks. @jjhelmus, @msarahan, any particular idea that you like?

Reply:
We can make a metapackage `cpython` and `pypy` to make it easier for the user to switch.

Reply:
(They wouldn't be a requirement of any package, but they are just there for users to easily remember. Just like `nomkl` is)